docs/core-concepts/core-sdk/cli/using-ci-with-cli.md

```diff
@@ -0,0 +1,58 @@
+# Using .NET Core SDK and tools in Continuous Integration
+
+## Overview
+This document outlines the usage of .NET Core SDK and its tools on the build server. In general, on a CI build server, 
+you want to automate the installation through some way. The automation, ideally, should not require administrative 
+privileges if at all possible. 
+
+For SaaS CI solutions, there are several options. This document will cover two very popular ones, [TravisCI](https://travis-ci.org/) and 
+[AppVeyor](https://www.appveyor.com/). There are, of course, many other services out there, but the installation and 
+usage mechanisms should be similar.
+
+## Installation options for CI build servers
+
+## Using the native installers
+If using installers that require administrative privileges is not something that bothers you, native installers for 
```

Comment:
is not something that bothers you -> is not a problem for you
if its not -> if it's not